examples/using-multiple-themes/src/gatsby-theme-blog/components/header.js

```diff
@@ -5,7 +5,7 @@ import Switch from "gatsby-theme-blog/src/components/switch"
 import Bio from "gatsby-theme-blog/src/components/bio"
 import sun from "gatsby-theme-blog/assets/sun.png"
 import moon from "gatsby-theme-blog/assets/moon.png"
-import Navigation from "../../components/navigation"
+import Navigation from "../../components/layout/navigation"
```

Comment:
This seems wrong. Why does the change to `www` affects `examples/using-multiple-themes`? Am I missing something?

Reply:
Thanks @vladar you are correct that is my mistake.